chronos/transports/stream.nim

```diff
@@ -1072,7 +1092,12 @@ when defined(windows):
                                              nil, flags)
           # Start tracking transport
           trackStream(ntransp)
-          server.createAcceptPipe()
+          try:
+            server.createAcceptPipe()
+          except CatchableError as exc:
+            # TODO close transp!
```

Comment:
closeHandle(sock) ?

Reply:
ok, pushed something, but take a look - the flow here is quite hard to follow